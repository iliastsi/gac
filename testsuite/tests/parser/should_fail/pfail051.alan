main () : proc
{
   writeString("This is a multi line string
		This program is wrong\n");
}
