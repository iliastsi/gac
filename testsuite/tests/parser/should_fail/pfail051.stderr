pfail051.alan:3:16: Error: Parse error on input `"'
	Cannot parse char `"'

pfail051.alan:3:22: Error: Parse error on input `is'
	Probably missing `)'

